### DISCUSSION TITLE
its stuck in this state... is it normal.. ?

### DISCUSSION THREAD

**Discussion by hilroamagi** [drizzle-mizzle/CharacterAI-Discord-Bot#107] [Q&A] [answered]
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/134889059/05864d88-7ec8-45ee-aa43-bd5c1440635c)

**Comment by Pyroturtle** [comment]
As far as I'm aware, that's normal
Unless it actually never loads
The wait times have been huge lately, and the site keeps on going down

**Comment by drizzle-mizzle** [comment, marked as answer]
Yeah
![WmweK6co_400x400.jpg](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/55811932/63588900-6f2e-4d49-9b82-7480c5c956f7)
